inspirehep/modules/authors/workflows/authornew.py

```diff
@@ -127,7 +127,7 @@ def formatter(bwo, **kwargs):
         else:
             # FIXME add a template for the author display in the HP
             return render_template("authors/workflows/authorupdate.html",
-                                   record_preview="",
+                                   record=bwo.data,
```

Comment:
The comments are fixed. I have just amended the commit. Cheers!

2015-12-01 14:27 GMT+01:00 Jan Åge Lavik <notifications@github.com>:

> In inspirehep/modules/authors/workflows/authornew.py
> <https://github.com/inspirehep/inspire-next/pull/587#discussion_r46276873>
> :
>
> > @@ -127,7 +127,7 @@ def formatter(bwo, **kwargs):
> >          else:
> >              # FIXME add a template for the author display in the HP
> >              return render_template("authors/workflows/authorupdate.html",
> > -                                   record_preview="",
> > +                                   record=bwo.data,
>
> I think you also need to do this change in
> inspirehep/modules/authors/workflows/authorupdate.py as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inspirehep/inspire-next/pull/587/files#r46276873>.
>